tests/Unit/Flowdock/FlowdockHandlerTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MonologHttp\Tests\Unit\Flowdock;
+
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Uri;
+use Monolog\Logger;
+use MonologHttp\Flowdock\FlowdockHandler;
+use MonologHttp\Tests\Unit\TestCase;
+use PHPUnit\Framework\MockObject\MockObject;
+use Psr\Http\Client\ClientInterface;
+use Psr\Http\Message\RequestFactoryInterface;
+use Psr\Http\Message\RequestInterface;
+
+class FlowdockHandlerTest extends TestCase
+{
+    /**
+     * @var ClientInterface|MockObject
+     */
+    private $client;
+
+    /**
+     * @var RequestFactoryInterface|MockObject
+     */
+    private $requestFactory;
+
+    /**
+     * @var FlowdockHandler
+     */
+    private $handler;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->client = $this->createMock(ClientInterface::class);
+        $this->requestFactory = $this->createMock(RequestFactoryInterface::class);
```

Comment:
I added the `guzzle/psr7: dev-master` which has a `RequestFactory` we can use that one instead of a mock here.